openspec/changes/add-non-interactive-init-options/specs/cli-init/spec.md

```diff
@@ -0,0 +1,74 @@
+# Delta for CLI Init Specification
+
+## ADDED Requirements
+### Requirement: Non-Interactive Mode
+The command SHALL support non-interactive operation through command-line options for automation and CI/CD use cases.
+
+#### Scenario: All tools selection
+- **WHEN** run with `--all-tools` option
+- **THEN** automatically select all available AI tools without prompting
+- **AND** proceed with initialization using the selected tools
+
+#### Scenario: Specific tools selection
+- **WHEN** run with `--tools <comma-separated-list>` option
+- **THEN** parse the comma-separated tool IDs and validate against available tools
+- **AND** proceed with initialization using only the specified valid tools
+- **AND** display a warning for any invalid tool IDs
+
+#### Scenario: No tools selection
+- **WHEN** run with `--skip-tools` option
+- **THEN** skip AI tool configuration entirely
+- **AND** only create the OpenSpec directory structure and template files
```

Comment:
What happens if someone tries to use `--all-tools` and `--skip--tools` at the same time? For CLI flags its better to try to keep each flag to one concern i.e one flag for tools. The below might be better/easier to simpler: `--tools all|none|a,b,c`